.mailmap

```diff
@@ -0,0 +1,5 @@
+Andrew Kofink <ajkofink@gmail.com> <akofink@redhat.com>
+Bernhard Hopfenmüller <hopfenmueller@atix.de> <bernhardh@atix.de>
```

Comment:
@Fobhep  please review which address you want as primary

Reply:
the first one i.e. hopfenmueller@atix.de